compiler/testData/codegen/boxInline/assert/jvmAssertInlineFunctionAssertionsEnabled.kt

```diff
@@ -1,3 +1,4 @@
+// IGNORE_BACKEND: JVM_IR
```

Comment:
Seems this could be supported via `IrInlineCodegen.generateAssertFieldIfNeeded`.
It's not required for this PR